tests/integration/test_tls.py

```diff
@@ -180,15 +187,37 @@ async def test_mtls(ops_test: OpsTest):
     # setting ACLs using normal sasl port
     await set_mtls_client_acls(ops_test, bootstrap_server=sasl_bootstrap_server)
 
+    num_messages = 10
+
     # running mtls producer
     action = await ops_test.model.units.get(f"{DUMMY_NAME}/0").run_action(
-        "run-mtls-producer", **{"bootstrap-server": ssl_bootstrap_server, "broker-ca": broker_ca}
+        "run-mtls-producer",
+        **{
+            "bootstrap-server": ssl_bootstrap_server,
+            "broker-ca": base64.b64encode(broker_ca.encode("utf-8")).decode("utf-8"),
```

Comment:
**nitpick:** I'm not sure aligning with the TLS operator pattern of:
```
- Start withstr
- utf8 encode it
- b64 encode it
- utf8 decode it  # I'm not sure this does anything
- b64 decode it
- utf8 decode it
- End with str
```

makes much sense. I don't see the value gained at all as it's a lot of forwards and backwards steps spread out across multiple files to reach the same point, is there a reason for it?

Also, what's the purpose of the utf-8 decode in the middle? On this line as the last method called. I don't think it does anything as it's b64 encoded at that point.

Reply:
Actually, this is not as relevant as the other modifications, but it was kinda easier (and similar to what done for tls-certificate-operator) to feed these inputs when using the charm "manually". 

I'd leave this as is given it does not seem a strong point